tv-shows-app/src/components/features/shows/ShowReviewSection/ShowReviewSections.utils.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  deleteAuthorizedMutator,
+  getAuthorizedMutator,
+  patchMutator,
+  postAuthorizedMutator,
+} from "@/app/fetchers/mutators";
+import { swrKeys } from "@/app/fetchers/swrKeys";
+import { IReviewFormInputs, IReviewList } from "@/typings/review";
+import { useState, useEffect } from "react";
+import useSWRMutation from "swr/mutation";
+
+export function useHandleReviews(id: string) {
+  const [reviewList, setReviewList] = useState<IReviewList>({ reviews: [] });
```

Comment:
We can start thinking about deleting states. SWR will keep the current state in cache for us.